pyprep/utils.py

```diff
@@ -480,19 +481,14 @@ def _mad(x, axis=None):
         the MAD for each index along the specified axis.
 
     """
-    # Ensure array is either 1D or 2D
```

Comment:
please do not just delete comments if they are unrelated to the changes

Reply:
yes i will fix all these in my commit